packages/ckeditor5-build-classic/sample/index.html

```diff
@@ -38,10 +38,32 @@ <h2>Sample</h2>
 	<p>You can use this sample to validate whether your <a href="https://ckeditor.com/docs/ckeditor5/latest/installation/advanced/alternative-setups/custom-builds.html">custom build</a> works fine.</p>
 </div>
 
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.6/MathJax.js?config=TeX-MML-AM_CHTML"></script>
+<script>
+      MathJax.Ajax.config.path["mhchem"] = "https://cdnjs.cloudflare.com/ajax/libs/mathjax-mhchem/3.3.2";
+      MathJax.Hub.Config({
+        showMathMenu: false,
+        TeX: {
+          extensions: [ "[mhchem]/mhchem.js" ]
+        },
+        messageStyle: "none",
+          tex2jax: {
+          preview: "none"
+        }
+      });
+</script>
+
 <script src="../build/ckeditor.js"></script>
 <script>
 	CKEDITOR.ClassicEditor.create( document.querySelector( '#editor' ), {
-		toolbar: ['heading', '|', 'bold', 'italic', 'link', 'underline', 'bulletedList', 'numberedList', 'blockQuote'],
+		math: {
+			engine: 'mathjax',
+			outputType: 'script',
+		},
+		toolbar: ['RemoveFormat', 'FontFamily', 'FontSize', 'FontColor', 'FontBackgroundColor', 'Underline', 'StrikeThrough',
+		'|', 'heading', 'Alignment',
+		'|', 'bold', 'italic', 'link', 'Strikethrough', 'underline', 'bulletedList', 'numberedList', 'blockQuote',
+		'|', 'SpecialCharacters', 'Math'],
```

Comment:
I wonder if all of these buttons will show up in the toolbar for CKEditor we use for Audioscript that Cambridge currently use?

Reply:
These don't have to be here in the sample.

Reply:
This config will only be used in sample. It is not exported or set as default in any case, so it will not have any impact on usage of exported instance CKEditor. It was used only for testing that needed plugins are enabled